README.md

```diff
@@ -35,21 +35,23 @@ workaround:
 
 ## How to install *realpath_turbo*
 
-1. Download the *realpath_turbo* source code.
-
-2. Extract and compile the extension
-
-   ```sh
-   $ tar -xaf realpath_turbo*
-   $ cd realpath_turbo*
-   $ phpize
-   $ ./configure
-   $ make
-   # make install
-   ```
-
-3. Adjust your `php.ini` to load and configure *realpath_turbo* extension.
-
+```sh
+wget -qO- https://github.com/Whissi/realpath_turbo/archive/master.tar.gz | tar -xz
+cd realpath_turbo-master/
+
+phpize
+./configure
+make
+make test NO_INTERACTION=1; echo $?
+make "INSTALL=$(pwd)/build/shtool install -c --mode=0644" install

```

Comment:
Picking up `build/shtool` shouldn't be necesseray -- configure will find the right install command.

Regarding the adjusted chmod: Note sure if the current chmod causes any problem. Anyway, that's a problem which should be addressed upstream. If we would adjust something we would have to make sure that we stay in sync.